ovirtlago/__init__.py

```diff
@@ -114,7 +114,7 @@ def prepare_repo(
 
         repos = []
 
-        if rpm_repo and reposync_yum_config:
+        if rpm_repo and os.path.isfile(reposync_yum_config):
```

Comment:
wouldn't it be cleaner to just ensure that reposync_yum_config gets initialized to None instead of a non-existent file name?
I think we do want to raise an error if the used specified a non-existent file on the command line...